lib/panel.js

```diff
@@ -84,6 +93,14 @@ class Panel {
 		failedContainer.textContent = assertResult.currentExecution.failed;
 	}
 
+	_getCssClassForAssert(assert) {
+		if (assert.ok) {
+			return assert.skip ? 'skipped' : 'ok';
+		}
+
+		return assert.todo ? 'todo' : 'ko';
```

Comment:
Should have been clearer. I'm aware how it works, but `ko` is a bad name. Should rather be `not-ok`, and updated in the LESS file too.